packages/ui/src/forum/components/ForumBreadcrumbs.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+import { BreadcrumbsItem } from '@/common/components/page/Sidebar/Breadcrumbs/BreadcrumbsItem'
+import { BreadcrumbsListComponent } from '@/common/components/page/Sidebar/Breadcrumbs/BreadcrumbsList'
+
+import { ForumBreadcrumb } from '../types'
+
+interface Props {
+  forumBreadcrumbs: ForumBreadcrumb[]
+  currentBreadcrumb: string
+}
+
+export const ForumBreadcrumbs = React.memo(({ forumBreadcrumbs, currentBreadcrumb }: Props) => {
+  const crumbs = forumBreadcrumbs.map((crumb) => ({
+    path: `/forum/categories/${crumb.id}`,
+    breadcrumb: crumb.title,
+    key: crumb.id + crumb.title,
```

Comment:
I'm not sure if that's needed for key. The `id` must be unique in the Query Node so I'd use `id` only.